src/xcode/ENA/ENA/Source/Views/GradientView.swift

```diff
@@ -154,6 +162,33 @@ class GradientView: UIView {
 			gradientLayer.locations = [0.25, 0.75]
 			gradientLayer.startPoint = CGPoint(x: 0.0, y: 0.0)
 			gradientLayer.endPoint = CGPoint(x: 1.0, y: 1.0)
+
+		case .certificateCheck:
+			let lightColors = [
+				UIColor(red: 255 / 255, green: 255 / 255, blue: 255 / 255, alpha: 1).cgColor,
+				UIColor(red: 255 / 255, green: 255 / 255, blue: 255 / 255, alpha: 1).cgColor,
+				UIColor(red: 235 / 255, green: 244 / 255, blue: 255 / 255, alpha: 1).cgColor
+			]
+
+			let darkColors = [
+				UIColor(red: 25 / 255, green: 25 / 255, blue: 27 / 255, alpha: 1).cgColor,
+				UIColor(red: 47 / 255, green: 65 / 255, blue: 77 / 255, alpha: 1).cgColor
+			]
```

Comment:
Should we define them in the ena colors?

Reply:
Probably not worth it as these are one-off colors just for the gradients